src/tools/wasm-split/wasm-split.cpp

```diff
@@ -385,6 +398,53 @@ void mergeProfiles(const WasmSplitOptions& options) {
   buffer.writeTo(out.getStream());
 }
 
+// todo: replace with better way converting to readable fn name
```

Comment:
```suggestion
// TODO: Make this unescaping optional with another flag.
```

Reply:
I am removing the todo as I added the unescape flag also